test/internal/options.go

```diff
@@ -24,6 +24,7 @@ import (
 // execution. Nil values represent no configuration of that type.
 type TOptions struct {
 	stopAfter         *TStopAfter
+	shouldError       *TShouldError
```

Comment:
Maybe consider expectedError rather than shouldError. It may make reading the code a bit smoother.

Reply:
Done